src/components/Content.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./Content.css";
+
+function Content() {
+  return (
+    <div>
+      <div class="mainContent">
+        <nav>
+          <img src="/images/ironhack-logo.svg"></img>
+          <img src="/images/menu-top.svg"></img>
+        </nav>
+        <div className="heroContainer">
+          <h1>Say hello to ReactJS</h1>
+          <h2>
+            You will learn how to use the most popular fronted library and
+            become super Ninja developer.
+          </h2>
+          <button>AWESOME!</button>
+        </div>
+      </div>
+      <div className="articleContainer">
+        <article>
+          <img src="/images/icon1.png"></img>
+          <h3>Declarative</h3>
+          <p>Random text</p>
+        </article>
```

Comment:
In here, since every "article" tag has the same structure, it would be a nice time to practice refactoring into a component! Maybe creating a component named "Article" and passing img src, the h3 and p value as props!